### ISSUE TITLE
Question: Mob spawn rules changed?

### ISSUE THREAD

**Issue by Teslachu** [NillerMedDild/Enigmatica2ExpertSkyblock#134]
Hey! So I play on a server and I have noticed that mobs spawn where they aren't supposed to. Like I am using the light overlay and making sure everything is lit up... I have made sure that any mob farms are a minimum of 20 blocks or more away from any other spawnable area. Now I know that somehow I am getting zombies to spawn on my top platform / base area, when my mob farm is at least (minimum) of 30 blocks away and it is completely lit up. I have no yellow / red squares.... No spawnable areas and yet this has happened at least 5 times today alone. 

Have spawn rules changed from the normal or am I just encountering some kind of weird bug? And before it is asked, my island is not seen as a village - I have no villagers or doors setup.

Now my mob farm does have cursed earth, but it that shouldn't be a problem.

**Comment by NillerMedDild** [comment]
We are aware of this issue, and we believe it is a bug. No modifications have been made to the spawning rules. We've unfortunately been unable to track down the bug, or even replicate it.

**Comment by GuntherDW** [comment]
Are you sure it isn't the "reinforcements" spawns from zombies that was introduced since I believe around Minecraft 1.7.0? If the difficulty is set to "hard" it will spawn other zombies as well.

https://minecraft.gamepedia.com/Zombie#Reinforcements

Turn the difficulty down if it's set to hard and check if it still happens.

**Comment by Teslachu** [comment]
Gunther, this is on a server. However, it is possible that the light level 9 requirement is being meet for them to call upon reinforcements. I will double check. Thanks for bringing that up.

**Comment by NillerMedDild** [comment]
Any news in regards to this issue?

**Comment by Teslachu** [comment]
My apologies. It does seem to be regarding the whole reinforcement rules. I must admit it is a bit weird though. I have played tons of sky blocks and this is the first one where I have had the issues. We did not change the difficulty setting, but we did watch carefully between an "automated" mob farm and a manual one. The manual one (where you go and kill them yourself) had less of that issue compared to the automatic one. Also watched where the zombies came from, when the automatic one was within range and it was from around a block that only had a light level of 9. So we moved the automatic one to a different dimension and kept the light level above 9 in all places except the farm itself - Sure enough - we have only had one issue (a skeleton and a spider) managed to be out of the farm... but they could have just been pushed out by some other mobs and it was right after setting the farm up... So ya... So far I would say it is the reinforcement spawning rule coming into effect. Like I said, I had never had that issue before, so it took me by surprise (though I did build with more slabs in other versions, so this would probably explain the difference).

TLDR: Probably not a game bug; I am almost positive that it is mob spawning rule of calling for reinforcements.

**Comment by NillerMedDild** [comment]
No worries, I'm glad you figured it out. Thank you for the help @GuntherDW, I hadn't thought of reinforcements at all :)
